src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface FigureSupplier extends ColorSupplier {
+    default Figure getRandomFigure() {
+        Random random = new Random();
+        int randomFigureType = random.nextInt(5);
+        int randomSide = random.nextInt() * 16;

```

Comment:
All magic numbers in your code should be constants.
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.